DQM/L1TMonitor/python/L1TStage2CaloLayer1_cfi.py

```diff
@@ -7,4 +7,6 @@
     hcalTPSourceSent = cms.InputTag("hcalDigis"),
     fedRawDataLabel  = cms.InputTag("rawDataCollector"),
     histFolder = cms.string('L1T/L1TStage2CaloLayer1'),
+    # HF minBias bit is not yet unpacked on HCAL side, so we don't compare them
```

Comment:
FYI @christopheralanwest here is where you can enable comparing the HF feature bit 2 when it is available from the HCAL unpacker

Reply:
The HF feature bit 2 is currently not available because of a uHTR FW bug, not a problem in the unpacker. Updated FW that includes a fix for this problem became available just yesterday, and is currently being tested. If the updated FW works as intended, this flag will become obsolete.

Reply:
Great news!
This is all ready to be merged so lets just put that in a new PR.